### PR TITLE
Docker Script Changes for working s3 sharedstorage demo

### PR DESCRIPTION
**Pull request by jmrojas2332**

### CODE REVIEW

File: scripts/cmapi-start [hunk 1]
```diff
@@ -8,6 +8,7 @@ API_KEY=${API_KEY:-somekey123}
 
 # Intialize Container If Necessary
 if [ ! -e $IFLAG ]; then
+  cp /opt/cmapi/engine_files/mcs-loadbrm.py /usr/bin/mcs-loadbrm.py
```
Comment (jmrojas2332) [inline scripts/cmapi-start]: Until new engine packages are ready, copy over the new mcs-loadbrm.py file shipped with cmapi.

File: scripts/cmapi-start [hunk 2]
```diff
@@ -22,9 +23,4 @@ echo CMAPI PID = $!
 # Start MariaDB
 /usr/share/mysql/mysql.server start
 
-# Start Columnstore Engine Proceses For Single Node
-if [[ $(hostname -s) == $PM1 ]]; then
-  curl -s -X PUT "https://$PM1:8640/cmapi/0.4.0/cluster/add-node" --header 'Content-Type:application/json' --header "x-api-key:$API_KEY" --data '{"timeout":60, "node": "'$PM1'"}' -k
-fi
-
```
Comment (jmrojas2332) [inline scripts/cmapi-start]: Observed some weirdness adding node automatically. Sometimes my machine would have one node, sometimes it didn't. Might have just been me. Feel free to ignore

File: scripts/columnstore-init [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 # Set Path Variables
 IFLAG=/etc/columnstore/container-initialized
+SHAREDFLAG=/var/lib/columnstore/storagemanager/sm-initialized
```
Comment (jmrojas2332) [inline scripts/columnstore-init]: Flag stored in sharedlocation if using shared volumes. Should not break system for non-shared volume usage as each node will only know of it's own /var/lib/columnstore/storagemanager folder if that volume is not shared

File: scripts/columnstore-init [hunk 4]
```diff
@@ -74,131 +75,133 @@ for i in $PROGS ; do
...
-   echo Sending SIGKILL to ColumnStore processes
-   kill -9 $(pidof $PROGS) > /dev/null
-fi
+  # Stop System
+  echo Stopping...
+
+  /usr/share/mysql/mysql.server stop
+
+  if  [[ ! -z $(pidof $PROGS) ]]; then
+    $MCS_INSTALL_BIN/mcs-savebrm.py
```
Comment (jmrojas2332) [inline scripts/columnstore-init]: should now be using mcs-savebrm.py

**Review by toddstoffel — DISMISSED** [review]

**Review by toddstoffel — COMMENTED** [review]
Already added.

**Pull request closed without merging**